portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> jsonValues;
+
+  @Override
+  public void init() {
+    jsonValues = new ArrayList();
+    jsonValues.add("This is the first message!");
+    jsonValues.add("Let's add another one!");
+    jsonValues.add("Well three's a crowd");
+    jsonValues.add("They said three was fine, but what's wrong with four.");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Eashan!</h1>");
+    String convertedJson = String.format("{\"firstMessage\": \"%s\", \"secondMessage\": \"%s\", " +
+                                         "\"thirdMessage\": \"%s\", \"fourthMessage\": \"%s\"}",
+                                         jsonValues.get(0), jsonValues.get(1), jsonValues.get(2), jsonValues.get(3));
```

Comment:
This line violates the 100 column limit as outlined in the style guide. Will Fix.